### PR TITLE
Fix for DistZip strip of artifact versions

### PR DESCRIPTION
**Pull request by vratsel**

### CODE REVIEW

**Comment by jesperpedersen** [comment]
Can one of the admins verify this patch?

**Comment by jesperpedersen** [comment]
ok to test

File: testsuite/src/test/java/org/ironjacamar/embedded/junit4/JUnit4TestCase.java [hunk 1]
```diff
@@ -68,6 +73,7 @@
    @Deployment(order = 1)
    private static URL deploymentStaticOne() throws Throwable
    {
+      System.out.println("deploymentStatic");
```
Comment (jesperpedersen) [inline testsuite/src/test/java/org/ironjacamar/embedded/junit4/JUnit4TestCase.java]: Remove this

File: testsuite/src/test/java/org/ironjacamar/embedded/junit4/JUnit4TestCase.java [hunk 2]
```diff
@@ -78,6 +84,7 @@ private static URL deploymentStaticOne() throws Throwable
    @Deployment(order = 1)
    private URL deploymentOne() throws Throwable
    {
+      System.out.println("deployment");
```
Comment (jesperpedersen) [inline testsuite/src/test/java/org/ironjacamar/embedded/junit4/JUnit4TestCase.java]: Remove this

**Comment by vratsel** [comment]
Updated

**Comment by jesperpedersen** [comment]
Merged, thanks for your contribution !

**Pull request closed without merging**